common/params.cc

```diff
@@ -93,6 +93,7 @@ std::unordered_map<std::string, uint32_t> keys = {
     {"CarVin", CLEAR_ON_MANAGER_START | CLEAR_ON_IGNITION_ON},
     {"CompletedTrainingVersion", PERSISTENT},
     {"ControlsReady", CLEAR_ON_MANAGER_START | CLEAR_ON_IGNITION_ON},
+    {"ControlsState", CLEAR_ON_MANAGER_START | CLEAR_ON_IGNITION_ON},
```

Comment:
Python doesn't allow you to set environment variables to bytes, had to make a param